src/entity/DevCard.ts

```diff
@@ -22,6 +33,15 @@ export class DevCard {
   @Column({ type: 'text', nullable: true })
   background: string | null;
 
+  @Column({ type: 'text', default: DevCardTheme.Default })
+  theme: DevCardTheme;
+
+  @Column({ type: 'bool', default: false })
+  isProfileCover: boolean;
```

Comment:
Just adding it here in case we want to do it that way, future proof - can we make this field `devCardCover` and it stores a string: either `default`, `profile cover` or `....url...` so we have the option for a new cover if we want to later to allow users to upload a new img.

But non blocking.